src/test/java/sysc4806/graduateAdmissions/model/TermTest.java

```diff
@@ -14,113 +19,135 @@
 
     private Term term;
     private Long id = 1L;
-    private String deadline = "deadline";
+    private Date deadline;
     private Season season = Season.WINTER;
     private String year = "year";
 
+    public TermTest() throws ParseException {
+        deadline = new SimpleDateFormat("yyyy-MM-dd").parse("2020-01-01");
+    }
+
 
     @BeforeEach
     void setUp() {
         term = new Term();
     }
 
+    /**
+     * Create a Term without arguments and verify correct defaults are set.
+     */
     @Test
-    /* Create a Term without arguments and verify correct defaults are set */
     public void testNoArgsConstructor() {
         assertNotNull(term);
-        assertNotNull(term.getId());
-        assertFalse(term.isActive());
+        assertNull(term.getId());
+        assertNull(term.getActive());
```

Comment:
Some of the Term instance variables are now objects so that we can check for validity and if they exist in the payload from the user. Having defaults assigned by the JVM when the user didn't specify something isn't always desirable.